src/axom/slam/policies/SetInterfacePolicies.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
```

Comment:
These copyrights will need to be updated. Sorry for letting this slip past the end of the year =/

Reply:
Hopefully, it should be sufficient to run this script: 
https://github.com/LLNL/axom/blob/develop/scripts/update_copyright_date.sh